src/lib/environment/mod.rs

```diff
@@ -533,6 +533,15 @@ pub(crate) fn setup_cwd(cwd: Option<&str>) {
                 "CARGO_MAKE_CARGO_HOME",
                 &home::cargo_home_with_cwd(directory_path).ok(),
             );
+            envmnt::set(
```

Comment:
i would change to set_or_remove so this would only be set if the crate_target_triple returns a value. otherwise we should remove it as i don't think 'unknown' is valid or useful and better just not have it.

Reply:
Done.

Reply:
I also moved the defaulting to the rust default target triple inside of `crate_target_triple`.

Reply:
Had to revert it again because it was setting it needlessly on every member, breaking all tests in the process.